packages/cozy-search/src/components/Conversations/styles.styl

```
@@ -1,18 +1,43 @@
+    border 0 !important
 
     +gt-mobile()
       max-height 178px
 
     &--mobile
-      border 0
       background-color transparent !important
       border-color transparent !important
 
+.conversationViewport
+  +mobile()
```

Stylelint (17.12.0)

[error] Unexpected newline before "+" (scss/operator-no-newline-before)

(scss/operator-no-newline-before)